### ISSUE TITLE
[BUG] UI broken after Update from 2.1.4 to 2.1.6

### ISSUE THREAD

**Issue by rstemmer** [Leantime/leantime#377]
**Describe the bug**
After updating my Leantime installation from 2.1.4 to 2.1.6 the UI has serious issues.
For example (most critical for me):
  * The Drag'n'Drop feature of the To-Dos (tickets)  board does not work anymore.
  * The Milestones-view is empty.
Looking at the browsers Consoles shows a flood of warnings an error messages.

_Firefox_
It starts with `Uncaught TypeError: jQuery.fn.dataTableExt is undefined` in tableHandling.js line 64
The errors and warnings are mostly about `$.browser is undefined` within some jQuery files.

_Chromium_
In chromium it starts with the following errors:
```
Uncaught TypeError: Cannot read property 'oApi' of undefined
    at tableHandling.js:76
showKanban:1614 Uncaught TypeError: Cannot read property 'initTicketSearchSubmit' of undefined
    at showKanban:1614
```
And then a first jQuery warning occures: `jQuery.Deferred exception: Cannot read property 'msie' of undefined TypeError: Cannot read property 'msie' of undefined` followed by lots of more and other warnings and errors in jQuery

**To Reproduce**
1. Go to 'leantime/tickets/showKanban'
2. Try to drag a To-Do tile

**Expected behavior**
The tile should be dragged and be placable within another column.
Instead only text gets marked as if there the tiles are not draggable.

**Leantime Version**
2.1.6 from the releases

**Server**
Apache/2.4.46

**PHP / MySQL Version**
PHP 7.4.14
mysql  Ver 15.1 Distrib 10.5.8-MariaDB

**Browser Version**
Firefox 84.0.2
Chromium Version 87.0.4280.141

**Comment by marcelfolaron** [comment]
Thanks for the bug. I'll investigate today and get a fix out as soon as possible

**Comment by yokazek** [comment]
Probably
`https://github.com/Leantime/leantime/releases/download/v2.1.6/Leantime-v2.1.6.tar.gz`
in
`public/js/compiled-base-libs.min.js`
and
`public/js/compiled-extended-libs.min.js`
are wrong.

```sh
npm update
. /node_modules/grunt/bin/grunt Build-All
```
If you do this, you should be fine.

**Comment by yokazek** [comment]
If you can't wait for the update, maybe you can replace it with this and it will work
public/js/compiled*

https://gist.github.com/yokazek/b649933808817b6604ad5a8991f72331

**Comment by marcelfolaron** [comment]
Should be fixed with 2.1.7
